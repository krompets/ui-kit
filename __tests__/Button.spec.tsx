import '@testing-library/jest-dom'
import { Button } from '@chakra-ui/react'

import { render } from 'jest.utils'

test('Button', () => {
  // expect(COMPONENTS).toBeTruthy()
  const rendered = render(<Button />)
  expect(rendered.container).toMatchSnapshot()
})
