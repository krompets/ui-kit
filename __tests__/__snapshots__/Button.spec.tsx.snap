// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button 1`] = `
<div>
  <button
    class="chakra-button css-54zu5w"
    type="button"
  />
</div>
`;
